### PR TITLE
Fix keyboard input handling

### LINKED ISSUE #57: Sheep movement scuffed
- [ ] Every key stroke results in jump when `UP` is pressed (inconsistend: `W` does not do the same)
- [ ] Moving feels terrible
When pressing left, then clicking and releasing right results in not moving at all.
(Different behavior with arrow keys)

### PR DESCRIPTION
**Pull request by Aloso**
Fixes #57

### CODE REVIEW

File: src/components/Sheep.tsx [hunk 1]
```diff
@@ -110,8 +116,40 @@ export const Sheep = ({
...
+        break;
+      case "d":
+        pressedKeys.d = true;
+        break;
+      case "ArrowLeft":
+        pressedKeys.left = true;
+        break;
+      case "ArrowRight":
+        pressedKeys.right = true;
+        break;
```
Comment (94726) [inline src/components/Sheep.tsx:131]: This whole ``pressedKeys.Key`` looks redundant to me. I personally would much rather have something like this:
 ```ts
rightMoveList = ['d', 'ArrowRight']
leftMoveList = ['a', 'ArrowLeft']
```

and with that, dynamically match states e. g.:
```ts
if (rightMoveList.includes(p5.key)) 
  isRight = true
  
if (leftMoveList.includes(p5.key)) 
  isLeft= true
```
And so on.

I can't really explain why I feel like that would be an improvement, it just seems like less duplication.
Maybe you and others can share their opinion about it

Comment (LeoTuet) [reply src/components/Sheep.tsx:131]: Yeah makes sense. Then it's also easier to add new keys for the controlling of the sheep in the future.

Comment (Aloso) [reply src/components/Sheep.tsx]: @94726 This is a good idea. Setting `isRight` or `isLeft` in the `keyPressed` function causes problems when multiple keys are pressed at the same time, but I managed to simplify the code. @LeoTuet I also moved all keyboard shortcuts into a single object, making the addition of new key bindings easier.

**Review by 94726 — APPROVED** [review]
Looks really good 👍

**Pull request merged**